packages/core/test/utils_test.js

```diff
@@ -2716,6 +2716,27 @@ describe("utils", () => {
       });
     });
 
+    it("should handle idSeparator parameter", () => {
+      const schema = {
+        definitions: {
+          testdef: {
+            type: "object",
+            properties: {
+              foo: { type: "string" },
+              bar: { type: "string" },
+            },
+          },
+        },
+        $ref: "#/definitions/testdef",
+      };
+
+      expect(toIdSchema(schema, undefined, schema, {}, "rjsf", "/")).eql({
+        $id: "rjsf",
+        foo: { $id: "rjsf/foo" },
+        bar: { $id: "rjsf/bar" },
+      });
+    });
+
```

Comment:
Thanks! One more thing. Can you add a test within Form_test.js that sets idSeparator and makes sure that the fields are rendered with the right idSeparator?

Reply:
@green7ea do you have time for adding this? Just one small change away from merging... if not, I can try